docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,99 @@
+## Math Formulae
+Markbind supports typesetting TeX math equations. [KaTeX](https://katex.org) is used as a fast math renderer.
+
+<box type="info">
+
+Insert `inline` equations by enclosing them in round brackets `\( ... \)`.
+
+Insert `display` equations by enclosing them in square brackets `\[ ... \]`.
+
+Insert numbered `display` equations by enclosing the equation square brackets and the equation number in curly brackets `\[ ... \] (1)`.
+</box>
+
+<box type="important">
```

Comment:
should we move this to the bottom? since the warning would make more sense after presenting the feature

Reply:
That makes sense; a user would want to know how to do things first before looking at its edge cases